CHANGELOG.md

```diff
@@ -4,7 +4,8 @@
 
 
 ## 1.7.0 in progress
-
+* [ENHANCEMENT] Added support for web indentity tokens
```

Comment:
Cortex release 1.8.0 is now in progress. Could you please rebase master and move the CHANGELOG entry under the master / unreleased section?